docs/index.html

```diff
@@ -1,8 +0,0 @@
-<!DOCTYPE html>
```

Comment:
this file was moved to scripts/docs/index.html. the docs folder is getting deleted entirely in a follow-up PR